tests/unit/field/test_field3d.cxx

```diff
@@ -644,11 +644,12 @@ TEST_F(Field3DTest, IterateOverRGN_NOY) {
   EXPECT_TRUE(region_indices == result_indices);
 }
 
-TEST_F(Field3DTest, IterateOverRGN_NOZ) {
+TEST_F(Field3DTest, IterateOverInvalid) {
```

Comment:
Just realised this means we're now not testing `RGN_NOZ` is it possible to add a test for this? Presumably just duplicating the `RGN_ALL` test.